chromium_presubmit_config.json5

```diff
@@ -156,7 +159,10 @@
       "components/speedreader/resources/third_party/",
       "components/webpack/gen-webpack-grd\\.js",
       "test/data/speedreader/",
-    ]
+    ],
+    "CheckNoJsInIos": ["ios/brave-ios/"],
+    "CheckForLongPathnames": ["third_party/ios_deps/"],
+    "CheckFilePermissions": ["third_party/ios_deps/"],
```

Comment:
pls move all these into `default_files_to_skip`.

Reply:
@goodov I want to eventually add an iOS app specific presubmit in the future, likely to run swiftlint, would adding these to `default_files_to_skip` cause them to be skipped there too or does this config file only affect Chromium specific presubmit checks?